### ISSUE TITLE
How to use it?

### ISSUE THREAD

**Issue by websharp950223** [MrH0wl/Cloudmare#37]
I am very happy to use your repo
but I get this issue while trying it
Virustotal probably now is blocking our requests! Or you have not a valid api key!
please let me know how to solve it

**Comment by MrH0wl** [comment]
Hi! That issue was solved a time ago by one of the contributors https://github.com/MrH0wl/Cloudmare/pull/31

Btw, that doesn’t imply any problem in the tool usage.

**Comment by websharp950223** [comment]
ok, let me try again

**Comment by MrH0wl** [comment]
To create an API for VirusTotal there is a mini-guide created by their developers: https://support.virustotal.com/hc/en-us/articles/115002100149-API

Just go to the “How to find your API key” section.

**Comment by websharp950223** [comment]
thanks for your attention
I just made the api key but I still got that issue
Could you please assist me?
My skype is topdev0223
thanks

**Comment by MrH0wl** [comment]
Have you already put the API key in the file /data/apis/api.conf?

Sorry, but can't assist you remotely right now, I'm not on my computer.

**Comment by websharp950223** [comment]
yes, I already did
but I got still issue
When could you please assist me?
if you fix that, it is very appreciated to me
What is your skype id?

**Comment by MrH0wl** [comment]
I don't have skype sorry, but if you could send me here the screenshots of the file “api.conf” (be sure of censoring your key and another screenshot of the tool running (be sure of censoring sensitive information).

**Comment by websharp950223** [comment]
here it 
![Screenshot_2](https://user-images.githubusercontent.com/93001796/141281219-23107cf2-6350-49a2-8cec-15e3c4ecfc4f.png)
is

**Comment by MrH0wl** [comment]
I see what the problem is now. It appears that the website has no subdomains to list. You can also use the -sC and/or -sSt arguments to have a more accurate result, if you can't get anything that means the website doesn't have a misconfiguration to obtain the IP. I'm closing this issue now, if you have any other questions, reopen it.
